src/Emulator/Visuals/Visuals.tsx

```diff
@@ -48,13 +48,26 @@ const HighlightButton: FC<BtnProps> = ({ children, hoverComponent, component, ..
 const Visuals = () => {
   const { sessionId } = useEmulator();
   const { data } = useVisualizeToynetImage(sessionId);
```

Comment:
`useVisualizeToynetImage` also returns a variable called `isLoading`. Is loading returns true if the request has not finished and false if it has. 

```suggestion
const { data, isLoading: isImageLoading } = useVisualizeToynetImage(sessionId);
```